package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/preset-react": "^7.16.5",
     "babel": "^6.23.0",
     "babel-loader": "^8.2.3",
+    "eslint": "^8.5.0",
```

Comment:
This is more of a note to myself to bring up to everyone tomorrow... we should albe using the same eslint rules now and running the linter before making PR.

Reply:
Ok! Let's discuss together.